src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -601,6 +602,8 @@ public String toString() {
                     SQLServerDriverStringProperty.DOMAIN.getDefaultValue(), false, null),
             new SQLServerDriverPropertyInfo(SQLServerDriverStringProperty.SERVER_NAME.toString(),
                     SQLServerDriverStringProperty.SERVER_NAME.getDefaultValue(), false, null),
+            new SQLServerDriverPropertyInfo(SQLServerDriverStringProperty.IPADDRESSPREFERENCE.toString(),
+                    SQLServerDriverStringProperty.IPADDRESSPREFERENCE.getDefaultValue(), false, null),
```

Comment:
The last param shouldn't be null right? It should be a string array containing the permissible IPAddressPreferences?

Reply:
Agreed, changes made.